docs/pages/distribution/release-channels.md

```diff
@@ -19,6 +19,8 @@ Publish your release by running:
 
 with the Expo CLI. Your users can see this release in the Expo Go app with a parameterized URL `https://exp.host/@username/yourApp?release-channel=<your-channel>`. If you do not specify a channel, you will publish to the `default` channel.
 
+A release channel name can only contain lowercase letters, numbers and special characters . _ and -.
```

Comment:
```suggestion
A release channel name can only contain lowercase letters, numbers and special characters `.`, `_` and `-`.
```

Maybe we can use inline code for the special characters readability?

Reply:
great idea